standard/classes.md

```diff
@@ -1382,6 +1384,68 @@ For a class containing a finalizer ([§15.13](classes.md#1513-finalizers)), the
 void Finalize();
 ```
 
+#### §Names-Reserved-For-Ops Member names reserved for operators
+
+The following method names are reserved. While many have corresponding operators in this specification, some are reserved for use by future versions, while some are reserved for interop with other languages.
```

Comment:
@KalleOlaviNiemitalo I'm not seeing a problem. I chose the new section heading based on the pattern of the previous 3 sections, "Member names reserved for ...," which contain only method signatures. Now as the set of operator-related method signatures is open-ended, I can't enumerate all those signatures, which is why I avoided using the term *signature* in the new section. I do say explicitly that these are method names, which of course are member names, so there is no conflict with the heading. Would you prefer the new section heading to say, "Method names reserved for operators"? If so, does that suggest the previous 3 headings also be changed to say "signatures" instead of "members"?

Reply:
I think the best resolution is to change the header from "member" to "method". I agree that enumerating all the signatures might be problematic.

Reply:
Yes, I think method name would be fine here.

Reply:
Change the header from "Member" to "Method".